lib/admin/WP_Auth0_Admin_Generic.php

```diff
@@ -133,7 +137,7 @@ protected function render_switch( $id, $input_name, $expand_id = '' ) {
 		printf(
 			'<div class="a0-switch"><input type="checkbox" name="%s[%s]" id="%s" data-expand="%s" value="1"%s>
 			<label for="%s"></label></div>',
-			esc_attr( $this->option_name ),
+			esc_attr( $this->_option_name ),
```

Comment:
Line 30 declares this
```
$options_name = $this->_option_name . '_' . strtolower( $id );
```

Why is this getting renamed to `_option_name` ? I'm not following that

Reply:
Renamed in a previous PR but these functions were added after that.